src/backend/mysql/query.rs

```diff
@@ -95,6 +95,10 @@ impl QueryBuilder for MysqlQueryBuilder {
         "()"
     }
 
+    fn random_function(&self) -> &str {
+        "RAND()"
```

Comment:
I don`t think brackets are needed here.

Reply:
I think they are! 

```
foursquaremaps=# select RANDOM;
ERROR:  column "random" does not exist
LINE 1: select RANDOM;
```

The parens need to be there and they're not added. The other example does work without parens:

```
foursquaremaps=# select CURRENT_TIMESTAMP;
       current_timestamp       
-------------------------------
 2022-09-01 16:38:11.467524+02
(1 row)
```

Reply:
Query generation does not insert the paren automatically and seems roundabout if it's a function with zero arguments.

Reply:
There's also `"(DEFAULT)"` where the parens are included in the output immediately.

Reply:
Ok, thank you for explain!

Reply:
Sadly, I think it's only because of `CURRENT_TIMESTAMP` which arguably is not a function